lib/views/components/recording/recording_content.dart

```diff
@@ -1,11 +1,39 @@
 
+import 'package:camera/camera.dart';
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../../../providers/video_recording_provider.dart';
 
 class RecordingPage extends StatelessWidget {
   const RecordingPage({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return Center(child: Text("im recording screen"),);
+    final cameraProvider = Provider.of<VideoRecordingProvider>(context);
+    return Scaffold(
+      appBar: AppBar(title: const Text('Video Recording')),
+      body: cameraProvider.isInitialized
```

Comment:
where is there an initialization?
if it not initialized then we should call the provider initializer wi future builder